src/main/java/com/strandls/observation/es/util/ObservationESDocument.java

```diff
@@ -105,6 +105,7 @@
 	private String location_scale;
 	private Long dataset_id;
 	private String dataset_title;
+	private String data_table_title;
```

Comment:
In query data_table_id, data_table_title, both are added, Here data_table_id is missing ??? why ?

Reply:
data_table_id has already been added

https://github.com/strandls/biodiv-observation/blob/f8cd18dc785051ae9572dcd193ace830815409d8/src/main/java/com/strandls/observation/es/util/ObservationESDocument.java#L117